packages/otelbin-recipes/recipes/test.yaml

```diff
@@ -0,0 +1,39 @@
+title: 'Foo'
+author: 'Bar'
+description: 'Hello World'
+config: |
```

Comment:
I feel we should use a similar markup as in [`supported-distributions.json`](https://github.com/dash0hq/otelbin/blob/08703d2b03e66df5cc9cef3d80c769d95f9264a9/packages/otelbin-validation/src/assets/supported-distributions.json#L51) to optionally allow-list distributions and version ranges

Reply:
TBH though I would love for us to have a recipe first that is not supported by all distros :)

Reply:
I guess a fair share of ADOT's wouldn't.

Also, any use of the `debug` extension breaks 0.81 (or so) and below.

Reply:
What would be the difference between the `invalid` state generated right now and an excluded range from the perspective of the user? Maybe I am missing something.

Reply:
One nice to have: tags on recipes could be super useful as the number of possible recipes grows so it's easy to filter, for example, by anything related to "aws" or "mysql", etc